### PR TITLE
feat: load preloaded upload files to use from a webview

### PR DESCRIPTION
**Pull request by Xmader**

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -178,36 +178,36 @@
...
+			{
+				mimeTypes: ['application/x-overture'],
+				extensions: ['.ove', '.scw'],
+				description: $t('overture_score_writer_files') + $t('experimental')
+			},
+			{
+				mimeTypes: ['application/x-bww'],
+				extensions: ['.bmw', '.bww'],
+				description: $t('bagpipe_music_writer_files') + $t('experimental')
+			},
```
Comment (Xmader) [inline src/routes/+page.svelte]: I don't think webmscore ever supported these file types

Comment (PeterNjeim) [reply src/routes/+page.svelte]: Does webmscore explicitly not support them? MuseScore supports them so that's why I added them

Comment (Xmader) [reply src/routes/+page.svelte]: https://github.com/LibreScore/webmscore/blob/v1.1.0/web-public/schemas.ts#L249-L257

Also in MuseScore 3 based version, it explicitly handles formats in https://github.com/LibreScore/MuseScore/blob/fd19dd923873148bf41dd2af60ccbc66de56e575/web/main.cpp#L125-L140

Comment (Xmader) [reply src/routes/+page.svelte]: Webmscore (MuseScore 4 based) currently register format handlers in https://github.com/LibreScore/webmscore/blob/v1.1.0/web/main.cpp#L143-L163

~~Of course I'll refactor the registration code later~~

Comment (PeterNjeim) [reply src/routes/+page.svelte]: MuseScore 4:
image

Comment (PeterNjeim) [reply src/routes/+page.svelte]: MuseScore backup files changed from `.mscz,` to `.mscz~`, I'll have to review their changes

Comment (Xmader) [reply src/routes/+page.svelte]: The official MuseScore 4 registers them here in their `main` function https://github.com/LibreScore/webmscore/blob/webmscore/src/main/main.cpp#L251-L262

I didn't use the code

Comment (Xmader) [reply src/routes/+page.svelte]: > MuseScore backup files changed from `.mscz,` to `.mscz~`, I'll have to review their changes

I found this
https://github.com/LibreScore/webmscore/blob/v1.1.0/src/project/internal/projectconfiguration.cpp#L659-L665

Comment (PeterNjeim) [reply src/routes/+page.svelte]: I'm guessing they changed the name due to the `,` not being supported in browsers: https://github.com/GoogleChromeLabs/browser-fs-access/issues/63

**Review by PeterNjeim — COMMENTED** [review]
No issues found

**Pull request merged**